subiquity/core.py

```diff
@@ -93,3 +93,10 @@ def _proxy_set(self):
             lambda fut: (
                 fut.result(), self.signal.emit_signal('snapd-network-change')),
             )
+
+    def unhandled_input(self, key):
+        if key == 'f1':
+            if not self.showing_help:
```

Comment:
AttributeError: 'Subiquity' object has no attribute 'showing_help'

Doesn't seem to be ever declared, hence pressing F1 crashes the installer.